src/NzbDrone.Common/Disk/TransferMode.cs

```diff
@@ -10,7 +10,10 @@ public enum TransferMode
         Move = 1,
         Copy = 2,
         HardLink = 4,
+        RefLink = 8,
 
-        HardLinkOrCopy = Copy | HardLink
+        HardLinkOrCopy = Copy | HardLink,
+        RefLinkOrCopy = Copy | RefLink,
+        RefLinkOrHardLinkOrCopy = Copy | RefLink | HardLink,
```

Comment:
I think these two are unused

Reply:
Removed and squashed where applicable for clean-up.